nghttp/README.md

```diff
@@ -0,0 +1 @@
+nghttp2/README.rst
```

Comment:
We render Markdown from readme on our web client, not sure how good will this particular rst file will look. I would suggest uploading this component to the staging first and checking the look

Reply:
I have sent staging server link for this component (in unicast). Rendering looked good to me, you may check as well.